modules/core/src/main/scala/Trace.scala

```diff
@@ -68,8 +78,10 @@ object Trace {
       Kleisli(_.put(fields: _*))
 
     def span[A](name: String)(k: Kleisli[F, Span[F], A]): Kleisli[F,Span[F],A] =
```

Comment:
As for naming: maybe this should be `spanned` and `spanR` could be the new `span`?

Reply:
btw. @tpolecat what do you think about having this as syntax on `F[A]`?

Reply:
Where `fa.span("foo")` = `Trace[F].span("foo")(fa)`? Seems useful.

Reply:
Yeah, just like that. I'll give a stab at these in different PRs then.